qt/DetailsDialog.h

```diff
@@ -89,20 +89,20 @@ private slots:
     void onIdleLimitChanged();
 
 private:
-    Session& mySession;
-    Prefs& myPrefs;
-    TorrentModel const& myModel;
+    Session& session_;
+    Prefs& prefs_;
+    TorrentModel const& model_;
 
     Ui::DetailsDialog ui;
```

Comment:
Looks like you didn't catch the cases which didn't use `myXxx` scheme before.

Reply:
tweaked in e5e8ffd2d91edb777af8564a885a4f2423c2043b

Reply:
That wasn't the only class with `ui` member. Didn't check if there're other (non-`ui`) cases out there.